src/components/SearchBar.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react'
+const SearchBar = ({setsearchData,searchData}) => {
+  const [searchText, setSearchText] = useState('')
+  const searchHandler = (searchText) => 
```

Comment:
can be returned without return key word and curly braces

Reply:
cool